module/state_synchronization/indexer/test_utils.go

```diff
@@ -0,0 +1,76 @@
+package indexer
+
+import (
+	"crypto/rand"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	"github.com/onflow/flow-go/ledger"
+	"github.com/onflow/flow-go/ledger/common/pathfinder"
+	"github.com/onflow/flow-go/ledger/complete"
+	"github.com/onflow/flow-go/utils/unittest"
+)
+
+// CreateTestTrieUpdate creates a test trie update with multiple test payloads
```

Comment:
we may have existing fixtures for these. check out https://github.com/onflow/flow-go/blob/master/utils/unittest/fixtures.go or search for the term "fixture". if not, can you add one using the existing framework we use so these can be used by other tests in the future.